### ISSUE TITLE
Doesn't really work with DC 9.62A pilot.bin

### ISSUE THREAD

**Issue by spth** [spth/OpenRabbit#14]
When using the pilot.bin from DC9.62A (which is shipped with OpenRabbit), openrabbitrfu does not give any error messages, but apparently, the user program is not written or not written correctly. I found this by using simple pin-setting examples (pe7 to 1, pe1 to 0 or the other way round):
When using OpenRabbit with DC9.62A pilot.bin, the pins still have the same state as before.

However, the very same programs work fine when either written using the RFU from DC 9.62A on Windows 10 or OpenRabbit with the pilot.bin from DC 8.61 (requires setting dc8pilot to true in rabbit.c).

**Comment by spth** [comment]
I made the Relase 0.2.1 to make it easier to use Openrabbit with pilot.bin from DC8, which is the workaround for this issue.

**Comment by spth** [comment]
Looks like the system info format changed from DC8 to DC9 pilot.bin:

With DC8 pilot.bin I get reasonably looking values:

~~~~
CPU:  0x0003
Freq: 11059200
sectorSize: 0x1000
numSectors: 0x0040
flashSize:  0x004
~~~~

With DC9 pilot.bin I see:

~~~~
CPU:  0x0020
Freq: 196663
sectorSize: 0x0001
numSectors: 0x0040
flashSize:  0xffffffff
~~~~

**Comment by spth** [comment]
Further investigation shows that pilot.bin from DC9 prepends an additional header to the reply to TC_SYSTEM_INFOPROBE requests. Apparently it consist of the four bytes 0x7c 0xfb 0x0f 0x00.
In the source of DC9 pilot, these four bytes are called "system id block".

**Comment by spth** [comment]
Fixed in 286db96 by skipping the system id block for parsing the info data.
